packages/react-frontend/src/pages/Playlist.jsx

```diff
@@ -37,62 +52,70 @@ function Playlist() {
   }
 
   function AddSong() {
-    const [song, setSong] = useState('');
+    const [song, setSong] = useState("");
 
     const handleInputChange = (event) => {
-        setSong(event.target.value);
+      setSong(event.target.value);
     };
 
     const handleAddSong = () => {
-        console.log('Adding song:', song);
-        
-        fetch("http://localhost:8000/playlists/" + path, {
-            method: 'POST',
-            headers: {
-                'Content-Type': 'application/json',
-            },
-            body: JSON.stringify({ name: song }),
-        })
-        .then(response => response.json())
-        .then(data => {
-            console.log('Success:', data);
-            fetchPlaylist()
-              .then(res => res.json())
-              .then(json => {
-                setPlaylist(json["playlist_list"]);
-              })
-              .catch(error => {
-                console.log(error);
-              });
+      console.log("Adding song:", song);
+
+      fetch("http://localhost:8000/playlists/" + path, {
+        method: "POST",
```

Comment:
We are going to have to change this, now that CI/CD is up and running. But this is a minor fix just need to change it to "https://songsmith.azurewebsites.net/playlists" + path